src/libs/fsdrv/lv_fs_littlefs.c

```diff
@@ -0,0 +1,191 @@
+#include "../../../lvgl.h"
+#if LV_USE_FS_LITTLEFS
+
+#include <lfs.h>
+#include "../../core/lv_global.h"
+
+struct LittleFile {
+    lfs_file_t file;
+};
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+static lfs_t lfs;
```

Comment:
Please move it to lv_logbal too.
You can add macro to make it easier to use.
```
#define lfs LV_GLOBAL_DEFAULT()->lfs

```

Reply:
As described [here](https://github.com/lvgl/lvgl/pull/5562#discussion_r1477777390), it think it should not be part of LVGL. If you need to pass a reference to `lfs`, we should a function like `lv_littlefs_set_handler()` that user can call before actually using LittleFS with LVGL.